log-collector-script/linux/eks-log-collector.sh

```diff
@@ -848,6 +848,34 @@ get_containerd_info() {
   fi
 
   ok
+
+  try "collect containerd snapshotter information"
+  get_soci_snapshotter_info
+
+  ok
+}
+
+get_soci_snapshotter_info() {
```

Comment:
I don't think the comments in this function are helpful. They are mostly 'what' comments that explain what the code is doing (which can be inferred by just reading the code), rather than pointing out complexities or reasons why you are doing something. I recommend deleting them.